include/common.hpp

```diff
@@ -4,13 +4,15 @@
 
 #include <cstdlib>
 #include <vector>
+#include <Eigen/Dense>
```

Comment:
add whitespace between different kinds of includes (std library vs external libraries)